sentry/templates/pvc.yaml

```diff
@@ -2,7 +2,7 @@
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: {{ template "sentry.prefix" . }}sentry-data
+  name: {{ template "sentry.prefix" . }}sentry
```

Comment:
Thanks ! I think we should call this `{{ template "sentry.prefix" . }}pvc`. And then we would need to change the deployment pvc value.

I would expect people to use `sentry-` as a prefix most of the time.

Can also update the version to 0.5.1, so that everyone can use your fix.

Thanks again!

Reply:
Where do you set this sentry.prefix??

Reply:
https://github.com/sentry-kubernetes/charts/blob/develop/sentry/templates/_helper.tpl#L6 here

Reply:
Thanks, I've updated PR.